core-bundle/tests/Framework/ContaoFrameworkTest.php

```diff
@@ -336,8 +335,6 @@ public function testRegistersTheHookServices(): void
 
     public function testServiceIsResetable(): void
     {
-        $this->assertInstanceOf(ResetInterface::class, $this->getFramework());
```

Comment:
Why?

Reply:
```
Call to method PHPUnit\Framework\Assert::assertInstanceOf() with 'Symfony\\Contracts\\Service\\ResetInterface' and
         Contao\CoreBundle\Framework\ContaoFramework will always evaluate to true.
         🪪  method.alreadyNarrowedType
```

Because `ContaoFramework` implements `ResetInterface`.